app/views/hyrax/collections/_list_collections.html.erb

```diff
@@ -3,7 +3,7 @@
   <td></td>
   <td>
     <div class="media">
-      <span class="<%= Hyrax::ModelIcon.css_class_for(::Collection) %> collection-icon-small pull-left"></span>
+      <span class="<%= Hyrax::ModelIcon.css_class_for(Hyrax.config.collection_class) %> collection-icon-small pull-left"></span>
```

Comment:
Should this be changed like the change in app/views/hyrax/collections/_media_display.html.erb?
```suggestion
      <span class="<%= collection_model_icon %> collection-icon-small pull-left"></span>
```